base/lnls-prune-artifacts.sh

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env bash
+
+set -Eeu
+
+filter_out_paths() {
```

Comment:
This function is missing a comment that its arguments should be split by newlines